src/libraries/System.Security.Cryptography/src/System/Security/Cryptography/HashProviderDispenser.Browser.cs

```diff
@@ -53,9 +53,15 @@ public static class OneShotHashProvider
 
             public static int HashData(string hashAlgorithmId, ReadOnlySpan<byte> source, Span<byte> destination)
             {
-                HashProvider provider = CreateHashProvider(hashAlgorithmId);
-                provider.AppendHashData(source);
-                return provider.FinalizeHashAndReset(destination);
+                switch (hashAlgorithmId)
```

Comment:
is CreateHashProvider dead code now?

Reply:
No, it is still being used by the non-one-shot hashes.